.github/workflows/release.yml

```
@@ -15,16 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        bin:
-          - pca-linux-amd64
-          - pca-linux-arm64
-          - pca-osx-amd64
-          - pca-osx-arm64
         include:
           - bin: pca-linux-amd64
             goos: linux
             arch: amd64
```

Please sort all of the `include` entries alphabetically by their keys:

```suggestion
          - arch: amd64
            bin: pca-linux-amd64
            goos: linux
```

I cannot do one full suggestion because the last two entries are out of range.

---

@mcdonnnj resolved with [bd30707](https://github.com/cisagov/con-pca-tasks/pull/7/commits/bd307077587166fe82227e78768b6f23b34d15ac)